2_adventures/001/amaliahern/04/fizz_buzz.rb

```diff
@@ -0,0 +1,6 @@
+def fizzbuzz(number)
+  return "fizzbuzz" if(number % 3 == 0 and number % 5 == 0)
```

Comment:
Wow! Thank you for your answer. Endeed I am going to work on this now. You cool!! :-D